agent/azure-monitor-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/implementation/livemetrics/LiveMetricsSpanProcessor.java

```diff
@@ -30,7 +31,6 @@ public boolean isStartRequired() {
   @Override
   public void onEnd(ReadableSpan readableSpan) {
     if (quickPulse.isEnabled()) {
-      // TODO (trask) can we do anything better here in terms of double conversion?
```

Comment:
not sure what this double conversion is referring to?

Reply:
I couldn't figure out either 😂 , I agree with deleting this comment 👍